src/Microsoft.DotNet.Core.Build.Tasks/GenerateDepsFile.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Extensions.DependencyModel;
+using Newtonsoft.Json;
+using NuGet.Frameworks;
+using NuGet.ProjectModel;
+
+namespace Microsoft.DotNet.Core.Build.Tasks
+{
+    /// <summary>
+    /// Generates the $(project).deps.json file.
```

Comment:
Can you suggest something better?  I felt this is the most straight forward way to describe this class.  And 99% of the time this will be true.